### DISCUSSION TITLE
Versioning and "support"

### DISCUSSION THREAD

**Discussion by daniloesk** [single-spa/single-spa-angular#327] [General]
Should the library community "worry" and "support" olders versions of Angular? For example, wouldn't it make sense to follow [Angular's support policy and schedule](https://angular.io/guide/releases#support-policy-and-schedule)? I started wondering this because of #325 , which has questions about Angular 6, which was released on May 3, 2018 and ended LTS support on November 3, 2019.

Of course, if anyone would like to help cases like this there is no problem at all, but if there was such s a policy inactive issues like this could be closed without much arguing.

**Comment by internettrans** [comment]
I think we should try to support older versions of Angular where we can, but that we don't have to spend a lot of time diagnosing issues with really old versions of Angular. A big reason people use single-spa is to help them migrate from older technologies to newer ones, so there are a fair number of people hoping for support of old versions. For example, I actively maintain single-spa-angularjs, which is for Angular 1, and a lot of groups still use it because they use single-spa as a migration strategy to be able to switch to a newer framework.

**Comment by daniloesk** [comment]
It's good to know about this important use case and have it clear that single-spa has a broader scope than Angular. Happy new year for all.
